src/internal/LabelBase.js

```diff
@@ -0,0 +1,92 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('LabelBase', (theme) => {
+  const focusColor = theme.palette.accent.A200;
+  return {
+    root: {
+      color: theme.palette.text.secondary,
+      lineHeight: 1,
+    },
+    focused: {
+      color: focusColor,
+    },
+    asterisk: {
+      // color: default color when unfocused
+    },
+    error: {
+      color: theme.palette.error[500],
+    },
+  };
+});
+
```

Comment:
Primary purpose is a reusable label with the asterisk/coloring behaviors for `required`, `focused`, and `error`